dnf-docker-test/features/module-defaults-conflict-1.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Fail if conflicting defaults
+
+  @setup
+  Scenario: Testing repository Setup
+      Given I run steps from file "modularity-repo-5.setup"
+       And I run steps from file "modularity-repo-8.setup"
+
+  @bz1656019
+  Scenario: Enabling two repository with conflicting defaults
+      When I enable repository "modularityConf"
+       And I successfully run "dnf module list"
+       And I enable repository "modularityConfConflict"
+       And I run "dnf module list"
+        Then the command should fail
+      When I successfully run "dnf module list --disablerepo=modularityConf"
```

Comment:
Is there anything missing? What is supposed to happen after the When part?

Reply:
Success means that dnf was not failing due to defaults merging issue

Reply:
It should have been "And" and not "When" then. :-)